pep-0303.txt

```diff
@@ -5,192 +5,203 @@ Last-Modified: $Date$
 Author: Thomas Bellman <bellman+pep-divmod@lysator.liu.se>
 Status: Rejected
 Type: Standards Track
-Content-Type: text/plain
+Content-Type: text/x-rst
 Created: 31-Dec-2002
 Python-Version: 2.3
 Post-History:
 
 
 Abstract
+========
+
+This PEP describes an extension to the built-in ``divmod()`` function,
+allowing it to take multiple divisors, chaining several calls to
+``divmod()`` into one.
 
-    This PEP describes an extension to the built-in divmod() function,
-    allowing it to take multiple divisors, chaining several calls to
-    divmod() into one.
 
 Pronouncement
+=============
+
+This PEP is rejected.  Most uses for chained ``divmod()`` involve a
+constant modulus (in radix conversions for example) and are more
+properly coded as a loop.  The example of splitting seconds
+into days/hours/minutes/seconds does not generalize to months
+and years; rather, the whole use case is handled more flexibly and
+robustly by date and time modules.  The other use cases mentioned
+in the PEP are somewhat rare in real code.  The proposal is also
+problematic in terms of clarity and obviousness.  In the examples,
+it is not immediately clear that the argument order is correct or
+that the target tuple is of the right length.  Users from other
+languages are more likely to understand the standard two argument
+form without having to re-read the documentation.  See python-dev
+discussion on 17 June 2005.
```

Comment:
Is this the correct discussion thread? https://mail.python.org/pipermail/python-dev/2005-June/054283.html
Should it be linked from here?